solrCloud_embedded_zooKeeper/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM --platform=$BUILDPLATFORM solr:8.11.2 AS indexer
```

Comment:
There are a few ways we could do this and keep all the solr files in the same directory.

1) Use a single directory for all the solr 8 stuff, but call this file `Dockerfile.embedded_zooKeeper`; reference from docker compose as

```yaml
build:
  context: ./solr8
  dockerfile: Dockerfile.embedded_zooKeeper
```

2) We could go farther and put everything in the same Dockerfile.

```dockerfile
FROM solr:8.11.2 AS common
# any steps everything needs
FROM common AS standalone
# any steps specific to building the standalone image
FROM common AS embedded_zookeeper
# any steps specific to building the image with embedded zookeeper
FROM common AS external_zookeeper
# any steps specific to building the image with external zookeeper
```

The in docker compose for example (varying `target` based on which one to use)

```yaml
build: 
  context: ./solr8
  target: embedded_zookeeper
```

See https://docs.docker.com/compose/compose-file/build/ for more info

Reply:
that is a great feedback! I will try to reproduce it!